src/Symfony/Component/Validator/ValidatorBuilder.php

```diff
@@ -35,13 +35,18 @@
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
-class ValidatorBuilder implements ValidatorBuilderInterface
+class ValidatorBuilder implements ExtraLoaderValidatorBuilderInterface
```

Comment:
I'd suggest moving this (allowing to add extra loaders to the validator) into another PR actually. I have a use-case already and it would be 👍 much quicker that part of this 😉

Reply:
Can we try to merge all in one? We still have time for 4.2 and this one is almost finished.